test/auth.js

```diff
@@ -97,4 +97,46 @@ describe('Authentication', () => {
             })
         })
     })
+    describe('/POST product', () => {
+        it('should fail because of missing authentication token', (done) => {
+            let product = {
+                name: "test_product",
+                brand: "test_product",
+                corporation: "test_corp",
+                barcode: "123456789",
+                state: "unverified"
+            }
+            chai.request(server)
+            .post('/product')
+            .send(product)
+            .end((err, res) => {
+                res.should.have.status(401);
+                res.body.should.have.property('message').eql("JWT is missing, access denied");
+                done();
+            })     
+        })
+    })
+    describe('/POST product', () => {
+        it('should fail because of manipulated authentication token', (done) => {
+            let product = {
+                name: "test_product",
+                brand: "test_product",
+                corporation: "test_corp",
+                barcode: "123456789",
+                state: "unverified"
+            }
+            chai.request(server)
+            .post('/product')
+            .set({ "Authorization": `Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFkbWluQHRlc3QuZGUiLCJpYXQiOjE2MTY1ODI3MjksImV4cCI6MTYxNzE4NzUyOX0.SXWuVkQu1fz0eCfLzN1E93TtZEYsoR60KB_UywiqIQ0`})
```

Comment:
Sure that you want to commit that Bearer-Token?

Reply:
this is a manipulated jwt as written in the test it's not valid